addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -4,121 +4,141 @@
+                                    "fields": ["address"],
+                                    "no_marker": True
+                                }'/>
+                                <div class="mb-3" t-field="event.address_id" t-options='{
+                                    "widget": "contact",
+                                    "fields": ["phone", "mobile", "email"]
+                                }'/>
+                                <a t-att-href="event._google_map_link()" target="_blank" class="btn btn-secondary text-bg-secondary w-100">
+                                    <p class="mb-0"><i class="fa fa-map-marker fa-fw" role="img"/>Get the direction</p>
+                                </a>
+                            </div>
+                            <!-- Organizer -->
+                            <div t-if="event.organizer_id" class="o_wevent_sidebar_block border-bottom pb-3 mb-3">
+                                <h6 class="o_wevent_sidebar_title text-muted text-uppercase">Organizer</h6>
+                                <h4 t-field="event.organizer_id"/>
+                                <div itemprop="location" t-field="event.organizer_id" t-options="{'widget': 'contact', 'fields': ['phone', 'mobile', 'email']}"/>
+                            </div>
+                            <!-- Social -->
+                            <div class="o_wevent_sidebar_block">
+                                <h6 class="o_wevent_sidebar_title text-muted text-uppercase">SHARE</h6>
+                                <p class="mb-2">Find out what people see and say about this event, and join the conversation.</p>
+                                <t t-snippet-call="website.s_share">
+                                    <t t-set="_no_title" t-value="True"/>
+                                    <t t-set="_classes" t-valuef="o_wevent_sidebar_social mx-n1"/>
+                                    <t t-set="_link_classes" t-valuef="o_wevent_social_link"/>
+                                </t>
+                            </div>
                         </div>
                     </div>
                 </div>
             </div>
         </section>
+        <t t-call="website_event.modal_ticket_registration"/>
     </t>
 </template>
 
 <!-- Event - Registration -->
 <template id="registration_template" name="Registration">
+    <div class=" d-flex d-lg-block flex-wrap justify-content-between align-items-center border-bottom">
+        <div class="d-flex flex-wrap justify-content-between align-items-center mb-3">
+            <h4 class="mb-0">Tickets</h4>
+            <t t-call="website_event.registration_configure_tickets_button" t-if="request.env.user.has_group('event.group_event_manager')"/>
+        </div>
+        <button t-if="event.event_registrations_open" type="button" data-bs-toggle="modal" data-bs-target="#modal_ticket_registration" class="btn btn-primary w-100">Register</button>
+    </div>
```

Comment:
So, this is the only part we actually show on mobile right?
I'd add a comment to specify that, it's not super obvious without being familiar with `d-lg-none`